extension/js/common/api/google.ts

```diff
@@ -286,8 +286,17 @@ export class Google extends Api {
       }
       return internalResults;
     },
-    fetchAtts: async (acctEmail: string, atts: Att[]) => {
-      const responses = await Promise.all(atts.map(a => Google.gmail.attGet(acctEmail, a.msgId!, a.id!)));
+    fetchAtts: async (acctEmail: string, atts: Att[], progressCb?: ProgressCb) => {
+      const loadedAr: Array<number> = [];
+      // 1.33 is a coefficient we need to multiply because total size we need to download is larger than all files together
+      const total = atts.map(x => x.length).reduce((a, b) => a + b) * 1.33;
+      const responses = await Promise.all(atts.map((a, index) => Google.gmail.attGet(acctEmail, a.msgId!, a.id!, (progress, loaded, s) => {
+        loadedAr[index] = loaded || 0;
+        const totalLoaded = loadedAr.reduce((a, b) => a + b);
+        if (progressCb) {
```

Comment:
This is the correct way to do it - check if `progressCb` exists instead of forcing it with `!`. Thanks!